diff --git a/themes/Gruntfile.js b/themes/Gruntfile.js
--- a/themes/Gruntfile.js
+++ b/themes/Gruntfile.js
@@ -28,6 +28,19 @@
         variables[key] = config.config[key];
     }

+    var ifaces = require('os').networkInterfaces();
+    var lookupIpAddress = null;
+    for (var dev in ifaces) {
+        if (dev != "eth0" && dev != "eth1") {
+            continue;
+        }
+        ifaces[dev].forEach(function (details) {
+            if (details.family == 'IPv4') {
+                lookupIpAddress = details.address;
+            }
+        });
+    }
+
     grunt.initConfig({
         uglify: {
             production: {
@@ -95,6 +108,23 @@
                 '../themes/Frontend/**/_public/src/js/*.js',
                 '../engine/Shopware/Plugins/**/frontend/**/src/js/**/*.js'
             ]
+        },
+        browserSync: {
+            dev: {
+                bsFiles: {
+                    src: [
+                        '../web/cache/*.css',
+                        '../web/cache/*.js'
+                    ]
+                },
+                options: {
+                    watchTask: true,
+                    reloadOnRestart: true,
+                    ws: true,
+                    host: lookupIpAddress,
+                    open: 'ui'
+                }
+            }
         }
     });

@@ -102,7 +132,8 @@
     grunt.loadNpmTasks('grunt-contrib-watch');
     grunt.loadNpmTasks('grunt-contrib-uglify');
     grunt.loadNpmTasks('grunt-contrib-jshint');
+    grunt.loadNpmTasks('grunt-browser-sync');

     grunt.registerTask('production', [ 'jshint', 'less:production', 'uglify:production' ]);
-    grunt.registerTask('default', [ 'less:development', 'uglify:development', 'watch' ]);
+    grunt.registerTask('default', [ 'less:development', 'uglify:development', 'browserSync', 'watch' ]);
 };
diff --git a/themes/package.json b/themes/package.json
--- a/themes/package.json
+++ b/themes/package.json
@@ -2,6 +2,7 @@
   "private": true,
   "devDependencies": {
     "grunt": "^0.4.5",
+    "grunt-browser-sync": "^2.2.0",
     "grunt-contrib-jshint": "^0.11.2",
     "grunt-contrib-less": "^1.0.1",
     "grunt-contrib-uglify": "^0.9.1",
